sphinxcontrib/trimblank.py

```diff
@@ -95,7 +95,7 @@ def _trim_blank(self, node):
                 self._logger.info(
                     '\nBefore : %s\nAfter  : %s',
                     child.astext(), new_txt, location=child)
-            node.replace(child, nodes.Text(new_txt, child.rawsource))
+            node.replace(child, nodes.Text(new_txt, child.astext()))
```

Comment:
Since docutils-0.18, "Text" nodes no longer have the rawsource attribute. So you don't need to pass it to the constructor. How about this?
```suggestion
            node.replace(child, nodes.Text(new_txt))
```

Reply:
Thank you for reviewing! Your suggestion is perfectly reasonable so I gladly accept it.

(For double check, I confirmed that the `rawsource` attribute was removed from Text nodes since docutils 0.18 according to its [release note](https://docutils.sourceforge.io/RELEASE-NOTES.html#release-0-18-2021-10-26). Also, I confirmed that although [`nodes.Text.__init__()`](https://sourceforge.net/p/docutils/code/HEAD/tree/tags/docutils-0.18.1/docutils/nodes.py#l408) accepts `rawsource` as the second argument, it does not use it. So, we can say removing the second argument to the constructor is perfectly safe.)